src/namer/namer.ts

```diff
@@ -78,6 +78,7 @@ async function process(session: Session<CodeModel>) {
   // pascal-case and capitzalize acronym operation groups and their operations
   for (const group of values(model.operationGroups)) {
     const details = <Language>group.language.go;
+    const opGroupName = capitalizeAcronyms(pascalCase(group.$key));
```

Comment:
What is `group.$key`?

Reply:
$key contains the operationGroup identifier

![image](https://user-images.githubusercontent.com/9859037/73579259-c1e2a400-4436-11ea-894e-2c52bb8d8593.png)